### PR TITLE
`FeatureFormView` - Revise example level error handling for attachments

### PR DESCRIPTION
**Pull request by dfeinzimer**
Related Apollo 679

### CODE REVIEW

**Comment by dfeinzimer** [comment]
@mhdostal Revised to simplify error display. This is ready for review.

File: Examples/Examples/FeatureFormExampleView.swift [hunk 1]
```diff
@@ -147,70 +168,204 @@ private extension URL {
...
+        case idle
+        /// A feature form is in use.
+        case editing(FeatureForm)
+        /// The form is being checked for validation errors.
+        case validating(FeatureForm)
+        /// Edits are being committed to the local geodatabase.
+        case finishingEdits(FeatureForm)
+        /// Edits are being applied to the remote service.
+        case applyingEdits(FeatureForm)
+
```
Comment (mhdostal) [inline Examples/Examples/FeatureFormExampleView.swift:183]: Was there a reason for the empty line?
```suggestion
```

File: Examples/Examples/FeatureFormExampleView.swift [hunk 2]
```diff
@@ -147,70 +168,204 @@ private extension URL {
...
         }
-        
-        try? await table.update(featureForm.feature)
-        
-        guard database.hasLocalEdits else {
-            print("No submittable changes found.")
+        if resultErrors.isEmpty {
+            state = .idle
+        } else {
+            state = .generalError(featureForm, Text("Changes were not applied."))
```
Comment (mhdostal) [inline Examples/Examples/FeatureFormExampleView.swift:321]: Maybe a comment here stating the client could display the errors to the user, if so desired.
```suggestion
            // Additionally, you could display the errors to the user using `resultErrors`.
            state = .generalError(featureForm, Text("Changes were not applied."))
```

**Review by mhdostal — COMMENTED** [review]
Just a couple of comments.

**Comment by mhdostal** [comment]
It is still pretty complicated, but the process of validating and applying changes is complicated as well, so I'm OK with it. We'll need to update the Tutorial as well.

File: Examples/Examples/FeatureFormExampleView.swift (hunk 1 above)
Comment (dfeinzimer) [reply Examples/Examples/FeatureFormExampleView.swift:183]: Just to form a separation between the linear states and the two others that can pop up at any time.

**Comment by dfeinzimer** [comment]
Tutorial updated, ready for re-review

**Review by mhdostal — APPROVED** [review]

**Pull request merged**